vendors/espressif/boards/esp32/CMakeLists.txt

```diff
@@ -137,11 +158,20 @@ target_include_directories(
 afr_mcu_port(wifi)
 
 if(NOT AFR_ESP_LWIP)
+if(ECC608_IN_USE)
+target_link_libraries(
+    AFR::wifi::mcu_port
+    INTERFACE
+    AFR::freertos_plus_tcp
+)
+else()
 target_link_libraries(
     AFR::wifi::mcu_port
     INTERFACE
     AFR::freertos_plus_tcp
 )
+endif()
```

Comment:
Are we covering the case of ECC608 + LwIP? Seems like that will end up being the default for this device in the near future.

Reply:
I will test that combination.

Reply:
Also this was a change I forgot to remove. I have done so now.

Reply:
I have now confirmed that this PR works for both LwIP and FreeRTOS+TCP.